### PR TITLE
345 Focus Area Profile View Bugs

### PR DESCRIPTION
**Pull request by cKellyDesign**
For #345

### CODE REVIEW

File: src/containers/pages/FocusInvestigation/single/index.tsx [hunk 1]
```diff
@@ -466,25 +466,25 @@ const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateP
...
       CASE_TRIGGERED
-    ),
+    ).filter(p => (jurisdiction ? p.jurisdiction_id === jurisdiction.jurisdiction_id : true)),
     currentRoutinePlansArray: getPlansArray(
       state,
       InterventionType.FI,
       [PlanStatus.ACTIVE, PlanStatus.DRAFT],
       ROUTINE
-    ),
+    ).filter(p => (jurisdiction ? p.jurisdiction_id === jurisdiction.jurisdiction_id : true)),
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/single/index.tsx]: I think we can do the filtering in the selector, ama?

Comment (cKellyDesign) [reply src/containers/pages/FocusInvestigation/single/index.tsx]: Good call

**Comment by moshthepitt** [comment]
@cKellyDesign do you think we'll have time to write a test for the selector change? Also wondering how the other change - for the map - can be captured in a test - should we create an issue for that one?

**Comment by cKellyDesign** [comment]
@moshthepitt let me try adding a quick test for the selector, not sure how to test for the Map changes so that should have a new ticket for that.

**Comment by cKellyDesign** [comment]
Test added (5200f4b), ticket created (https://github.com/onaio/reveal-frontend/issues/362)

**Pull request merged**